concepts/05 Widgets/CheckBox/00 Overview.md

```diff
@@ -25,14 +25,14 @@ The following code adds the **CheckBox** to your page.
     <!--HTML-->
     <dx-check-box
         text="Check me"
-        [(value)]="checkBoxValue">
+        [(value)]="value">
```

Comment:
Please give this variable a name that reflects its purpose. Why did you decide to rename it?

Reply:
fixed